flink-table/flink-sql-client/src/test/java/org/apache/flink/table/client/SqlClientTest.java

```diff
@@ -38,36 +37,35 @@
 import java.net.URL;
 import java.nio.charset.StandardCharsets;
 import java.nio.file.Files;
+import java.nio.file.Path;
 import java.nio.file.Paths;
 import java.nio.file.StandardOpenOption;
 import java.util.Arrays;
 import java.util.Collections;
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
-import java.util.concurrent.TimeUnit;
 
 import static org.apache.flink.configuration.ConfigConstants.ENV_FLINK_CONF_DIR;
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.assertj.core.api.Assertions.assertThatThrownBy;
 
 /** Tests for {@link SqlClient}. */
-public class SqlClientTest {
+@Timeout(value = 1000)
```

Comment:
Since JUnit 5 supports specification of timeunits it would be much more convenient to use them like 
```java
@Timeout(value = 1, unit = TimeUnit.SECONDS)
```
or
```java
@Timeout(value = 1000, unit = TimeUnit.MILLISECONDS)
```